lib/model.js

```diff
@@ -110,9 +110,13 @@ internals.Validator = class SchwiftyValidator extends Objection.Validator {
         }
 
         const validation = ctx.joiSchema.validate(json);
+        //console.log('JUST VALIDATED', validation);
 
         if (validation.error) {
             const info = internals.parseJoiValidationError(validation);
+
+            // constructor must be passed object of schema
```

Comment:
Just a nano-FYI 📚sometimes folks also call this the "shape" of the object.  I like geometry :) 📐